server/src/infra/repositories/media.repository.ts

```diff
@@ -87,7 +88,10 @@ export class MediaRepository implements IMediaRepository {
         ffmpeg(input, { niceness: 10 })
           .outputOptions(options.outputOptions)
           .output(output)
-          .on('error', reject)
+          .on('error', (err, stdout, stderr) => {
+            Logger.error(stderr);
```

Comment:
Can you instantiate a new logger with a name instead of using the static instance?